adr/base-protocols.md

```diff
@@ -0,0 +1,40 @@
+# Baseline protocols
+
+**Authors:**
+
+- Leif Johansson, Sunet, Sweden
+- Sander Dijkhuis, Cleverbase, the Netherlands
+- Sarah Amandusson, Digg, Sweden
+
+## Context
+The EUDI Wallet ecosystem is mandated by eIDAS to ensure secure, interoperable cross-border digital identity.
+This implementation is guided by the Architecture and Reference Framework (ARF), which translates legal mandates into technical specifications.
+To achieve mandatory interoperability for Issuance and Presentation of Person Identification Data (PID) and Electronic Attestations of Attributes (EAA), implementing regulations require compliance with foundational standards.
+OID4VCI (Issuance) and OID4VP (Presentation) are adopted as the ARF-recommended baseline protocols for technically implementing the required data models and web-based transport mechanisms in the pilot.
+
+Proximity flows are out of scope for the current architecture decision.
+
+While the architecture decision to [Publish consortium trusted lists](trusted-lists.md) based on TS 119 612 has already been recorded, the consortium also needs a selection of PID/EAA issuance and presentation protocols.
```

Comment:
the impl profile is proposed in the PR below: https://github.com/webuild-consortium/wp4-trust-group/pull/13

please note that it must be updated to TS version 2.4, the trust group in on that.